airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/extensions/PathExtensions.kt

```diff
@@ -0,0 +1,22 @@
+package io.airbyte.cdk.extensions
+
+import java.nio.file.Path
+import java.nio.file.attribute.PosixFilePermission
+import kotlin.io.path.setPosixFilePermissions
+
+fun Path.grantAllPermissions(): Path {
+    this.setPosixFilePermissions(
+        setOf(
+            PosixFilePermission.OWNER_READ,
+            PosixFilePermission.OWNER_WRITE,
+            PosixFilePermission.OWNER_EXECUTE,
+            PosixFilePermission.GROUP_READ,
+            PosixFilePermission.GROUP_WRITE,
+            PosixFilePermission.GROUP_EXECUTE,
+            PosixFilePermission.OTHERS_READ,
+            PosixFilePermission.OTHERS_WRITE,
+            PosixFilePermission.OTHERS_EXECUTE,
+        ),
+    )
+    return this
+}
```

Comment:
These are duplicated because the two test suites share nothing (by design?) The other classes in this package are similarily duplicated.